src/ZoneServer/Buffs/Handlers/Scout/Assassin/Behead_Debuff.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Network;
+using Melia.Zone.Skills;
+using Melia.Zone.Skills.Combat;
+using Yggdrasil.Logging;
+
+namespace Melia.Zone.Buffs.Handlers.Scout.Assassin
+{
+	/// <summary>
+	/// Buff handler for Behead: Bleeding, which deals damage
+	/// every 1.5 seconds
+	/// </summary>
+	[BuffHandler(BuffId.Behead_Debuff)]
+	public class Behead_Debuff : BuffHandler
+	{
+		public override void WhileActive(Buff buff)
+		{
+			if (!buff.Target.IsDead) { 
+				var skill = new Skill(buff.Caster, SkillId.Assassin_Behead_DOT, (int)buff.NumArg1);
```

Comment:
It feels kinda iffy to create a new throwaway Skill object every tick, doesn't it? Especially because we know this would quickly get copied to all buffs and skills with similar effects. Do officials actually send `Assassin_Behead_DOT`?

Reply:
I added an overload to ZC_Hit_Info that takes a SkillId instead of a Skill object (it only uses the SkillId anyway), which allows this to work without creating a skill object for every tick.